src/entries/popup/pages/send/ReviewSheet.tsx

```diff
@@ -332,12 +331,12 @@ export const ReviewSheet = ({
     useExplainerSheetParams();
 
   const showL2Explainer = useCallback(() => {
-    if (!asset?.chainId || !isSideChain(asset.chainId)) return;
+    if (!chain?.id || !isSideChain(chain?.id as ChainId)) return;
     showExplainerSheet({
-      ...getSideChainExplainerParams(asset.chainId, hideExplainerSheet),
+      ...getSideChainExplainerParams(chain?.id as ChainId, hideExplainerSheet),
```

Comment:
this doesn't need to be optionally chained if you're null checking beforehand. Also, any way to avoid having to cast `as ChainId`? I kinda hate this pattern of casting `number` -> `ChainId`

Reply:
yes. will fix this up later today.